test/set/set-iteration/test-2-spec.k

```diff
@@ -14,5 +14,8 @@ module TEST-2-SPEC
 
   rule <k> runTest( f( SetItem(X:Element) SetItem(Y:Element) ) ) </k>
     => <k> doneTest( SetItem(g(X)) SetItem(g(Y)) ) </k>
+    requires
+      (X =/=K Y)
+      andBool (g(X) =/=K g(Y))
```

Comment:
Why is this needed?

Reply:
This is needed to ensure the set is defined. Before, when we used matching, this condition didn't need to be true.